technobly/vextreme
# Adds HW and SW versions to VEXTREME menu and Vectrex games

### Problem

See #34 and #26 

### Solution

- Adds the SW version to system options
- Patches the multcart.bin upon loading to force HW and SW version info to be displayed in the game header.
- Adds a RPC function ID 9 that allows a game to read in HW and SW revisions into the last 4 bytes of 32K ROM.
- Adds a demo of RPC function ID 9 in LED Test v1.1

### Steps to Test

- Compile and flash veccart.bin, multicart.bin, led-test1.bin
- Run the above and observe new version info!

### References

Fixes #34 
Fixes #26 

![image0](https://user-images.githubusercontent.com/2249224/79033988-4dd02500-7b78-11ea-942b-0bb2ec98babb.jpg)
